.github/workflows/activate-docker-distribution.yaml

```diff
@@ -62,20 +62,24 @@ jobs:
           docker manifest inspect ${{ env.DESTINATION_IMAGE_TAG }}
           EXIT_CODE=$?
           if [[ $EXIT_CODE -eq 0 ]]; then
-            echo "ERROR: image ${{ env.DESTINATION_IMAGE_TAG }} already exist"
+            echo "::error:: image ${{ env.DESTINATION_IMAGE_TAG }} already exist"
             exit 1
-          elif [[ $EXIT_CODE -ne 1 ]]
-            echo "ERROR: $EXIT_CODE"
+          elif [[ $EXIT_CODE -ne 1 ]]; then
+            echo "::error:: $EXIT_CODE"
             exit 1
           fi
 
       - name: "Update image tag"
         run: |
           set +e
+          docker pull ${{ env.SOURCE_IMAGE_TAG }}
           docker image tag ${{ env.SOURCE_IMAGE_TAG }} ${{ env.DESTINATION_IMAGE_TAG }}
-          if [[ "${{ inputs.set_latest }}" ]]; then
+          
+          if [[ ${{ inputs.set_latest }} ]]; then
             docker image tag ${{ env.SOURCE_IMAGE_TAG }} latest
           fi
-          if [[ "${{ inputs.delete_source }}" ]]; then
-            docker rmi ${{ env.SOURCE_IMAGE_TAG }}
+          docker push
+          if [[ ${{ inputs.delete_source }} ]]; then
```

Comment:
I know Stefan disagres, I think this is super bad practice to be altering the history like this.